### ISSUE TITLE
[Screen Reader-Dual Write-Customize already created Entity] In scan mode, focus of the Screen Reader moves out of the "Contactpersonpartynumbe"dialog box.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#11686]
#URL:https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. 
“HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.

Please reach out to C&AI Teams channel for any process related queries.

**User Experience:** 
If, in scan mode focus of the Screen Reader moves out of the dialog box, then the user who depends upon Screen Reader for navigation might get affected hence failed to navigate properly on the controls available on the dialog box.
**Note:** User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch the URL: https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com 
2. Sign in with Valid credentials.
3. Dual Write entity Maps screen appearing.
4. Navigate to any entity and select context menu button.
5. Navigate to "Entity mapping" and select it .
6. Navigate to already created entity mapping  and select "Map type" eg."=".
7. New screen appearing as "Contactpersonpartynumber".
8. Navigate with the Narrator in scan mode on the upon on the dialog box.
9. Verify, whether in scan mode focus of Screen Reader does move out of the dialog box or not.

**Actual Result:**
On the Contactpersonpartynumber dialog box, in scan mode focus of the Screen Reader moves out of the dialog box. Focus of the Screen Reader moves to background controls.

**Expected Result:**
On the Contactpersonpartynumber dialog box, in scan mode focus of the Screen Reader should remain inside the dialog box. So, that the Screen Reader user does able to navigate properly on the controls of dialog box.

**Note:** 1. This issue does exist with Chrome +JAWS (in cursor mode).
          2. This issue exist throughout the application for all the dialog box.

**Test Environment:**
OS: Windows 10, Version 1903 (OS build 18960.1000)
Browser: Chrome, Edge (44.18895.1000.0)
Screen Reader: Narrator, JAWS
[URL:https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com]

**MAS Reference:**
MAS 4.2.10 - Tracking of Focus and Selection Attributes

**Attachment :**
[MAS4.2.10-In scan mode focus of Screen reader moves out of dialog box.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/4053270/MAS4.2.10-In.scan.mode.focus.of.Screen.reader.moves.out.of.dialog.box.zip)

Bug copied from Dual write : https://msazure.visualstudio.com/OneAgile/_workitems/edit/5708240
